src/UI/Seller/src/app/seller-users/seller-user.service.ts

```diff
@@ -32,4 +35,23 @@ export class SellerUserService extends ResourceCrudService<User> {
     Email: '',
     Phone: '',
   }
+
+  async createNewResource(resource: any): Promise<any> {
+    try {
+      const args = await this.createListArgs([resource])
+      const newResource = await this.ocService.Create(...args)
+      this.resourceSubject.value.Items = [
+        ...this.resourceSubject.value.Items,
+        newResource,
+      ]
+      this.resourceSubject.next(this.resourceSubject.value)
```

Comment:
It looks like we are setting the resourceSubject.value and then nexting the resourceSubject with it's own value.. I'm assuming this was necessary, but i'd think you can just do this.resourceSubject.next({append newResource here}]). Could you talk me though how passing the resourceSubject's value to itself works?

Reply:
I can try to remove that line, but I'll explain what my process here was.  I just needed to override this function from the `resource-crud.service.ts` in order to intercept errors to throw custom messaging via toastr.  So, I literally copied and pasted this function, which I did not initially author, to this file and made my overrides.  I will try removing that line and get back to you here about if it works or not.

Reply:
So it looks like the creation went off without a hitch, so that must have been a misunderstanding on the part of the original author?  I will push the change with it removed.

Reply:
ah, that makes sense. thanks for looking into it! From my vantage point, if you are using/copying the code, it should be understood how it's working since now you are the "author" of the copied code. (obviously you know how it's working so it's not a concern, just so you know where my head is at on that topic!).

Reply:
Definitely, I appreciate the feedback!